### ISSUE TITLE
Use logzero as root_logger.

### ISSUE THREAD

**Issue by fx-kirin** [metachris/logzero#144]
* logzero version:1.5
* Python version:3.6
* Operating System:Linux

### Description

As I wanted to use parent logger of each setup_logger instance, I tried to find the way to do it.

Buf default logzero setting is not what I intend to use for. The default is not inherited any instance of 'setup_logger'.

### What I Did

```
import logzero
logzero.__name__ = ''
logzero.setup_logger()
```
When I executed the code above, I've used it as root_logger. And stopped using `setup_logger`. I used logger.get_logger('name') instead. Then, finally I got it work as I supposed to do.

### Suggestion

I think it's not bad idea to describe how to use logzero as rootlogger in README or make a method to do this. I just would like to leave it in case someone else try to do the same thing.

**Comment by metachris** [comment]
Thanks for the report. #342 is a possible solution. Thoughts?

**Comment by metachris** [comment]
Fixed in master. Will update this issue after a release

**Comment by metachris** [comment]
Fixed in v1.6.2
